airflow/api_connexion/openapi/v1.yaml

```diff
@@ -1720,6 +1720,65 @@ paths:
         "404":
           $ref: "#/components/responses/NotFound"
 
+  /dags/{dag_id}/dagRuns/{dag_run_id}/taskInstances/{task_id}/tries/{task_try_number}:
+    get:
+      summary: get taskinstance try
+      description: |
+        Get details of a task instance try.
+
+        *New in version 2.10.0*
+      x-openapi-router-controller: airflow.api_connexion.endpoints.task_instance_endpoint
+      operationId: get_task_instance_try_details
+      tags: [TaskInstance]
+      parameters:
+        - $ref: "#/components/parameters/DAGID"
+        - $ref: "#/components/parameters/DAGRunID"
+        - $ref: "#/components/parameters/TaskID"
+        - $ref: "#/components/parameters/TaskTryNumber"
+      responses:
+        "200":
+          description: Success.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/TaskInstance"
+        "401":
+          $ref: "#/components/responses/Unauthenticated"
+        "403":
+          $ref: "#/components/responses/PermissionDenied"
+        "404":
+          $ref: "#/components/responses/NotFound"
+
+  /dags/{dag_id}/dagRuns/{dag_run_id}/taskInstances/{task_id}/tries/{task_try_number}/{map_index}:
```

Comment:
Probably makes sense to put map_index farther up the path? Maybe `/dags/{dag_id}/dagRuns/{dag_run_id}/taskInstances/{task_id}/{map_index}/tries/{task_try_number}`?

That matches the other mapped ti endpoint better.

Reply:
This would make the mapped_task try endpoint different from the normal task try endpoint

Reply:
Not sure I follow what you mean. Here are the existing endpoints:

```
get:         /dags/{dag_id}/dagRuns/{dag_run_id}/taskInstances/{task_id}
get mapped:  /dags/{dag_id}/dagRuns/{dag_run_id}/taskInstances/{task_id}/{map_index}
```

With my proposed change, you'd just add the `/tries/{try_number}` to those above: 

```
history:         /dags/{dag_id}/dagRuns/{dag_run_id}/taskInstances/{task_id}/tries/{task_try_number}
history mapped:  /dags/{dag_id}/dagRuns/{dag_run_id}/taskInstances/{task_id}/{map_index}/tries/{task_try_number}
```

What I like about this is:
- It keeps the PK of the TI together, vs putting the try number in the middle
- You're just adding more to the path for getting try details, vs having a different path altogether

Reply:
I got it. I was talking about the `try` endpoints for both mapped and normal being similar, but I see what you mean: get_mapped being similar to get_mapped_history. I will update it with the change.